src/lib/viewers/BaseViewer.js

```diff
@@ -966,6 +966,9 @@ class BaseViewer extends EventEmitter {
      * @return {void}
      */
     initAnnotations() {
+        // Annotations must be loaded and ready before scrolling can occur on a deep-linked annotation.
+        this.annotator.addListener('annotations_ready', this.handleAnnotationsOnLoad);
```

Comment:
I think @mxiao6 is adding a `annotations_fetch_error` event, would `annotations_fetch_success` make more sense?

Reply:
Can we keep our terminology consistent? We're using `init`, `ready`, and `load` interchangeably.